components/motor/errors.go

```diff
@@ -11,7 +11,7 @@ func NewResetZeroPositionUnsupportedError(motorName string) error {
 // NewPropertyUnsupportedError returns an error representing the need
 // for a motor to support a particular property.
 func NewPropertyUnsupportedError(prop Properties, motorName string) error {
-	return errors.Errorf("motor named %s must support property motor.%v", motorName, prop)
+	return errors.Errorf("motor named %s has wrong support for property %#v", motorName, prop.PositionReporting)
```

Comment:
I don't get the phrasing on this. I expect it will give errors like "motor named front_left has wrong support for property false" and I can't tell what that means. What's the error message supposed to look like?

Reply:
I agree, would it make sense to pass in a string or something else instead of a property?

Reply:
there could be many properties, right now it renders as:

`"motor named left must support property motor.Properties{false}"`

I am making it render as:
`error motor named left has wrong support for property  motor.Properties{PositionReporting:false}`

Reply:
Huh, I must be doing something wrong in [the Go playground](https://go.dev/play/p/UTZwF1GKdZp), since I'm just getting it rendered as `false`. but if you've been able to generate this error and it looks right to you, then the rest LGTM. :shrug:

Reply:
no you're right - I pushed the wrong thing up